djangocms_navigation/static/djangocms_navigation/js/navigation-tree-admin.js

```diff
@@ -0,0 +1,367 @@
+(function ($) {
+        ACTIVE_NODE_BG_COLOR = '#B7D7E8';
```

Comment:
Appears to be double indented. Some FE files are 2 spaces as per most guidelines, I'm not particularly bothers as long as the files in a specific project are consistent.

Reply:
Well spotted! Indentation in this file is four spaces - I've updated and made consistent throughout.